jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e.yaml

```diff
@@ -1164,3 +1163,42 @@
             version-infix: '1-2-1-3'
             version-env: ''
             legacy-ginkgo-test-args-env: ''
+
+# Template for Docker continuous validation tests, running on GCI. See
+# https://github.com/kubernetes/kubernetes/issues/25215 for details of what the
+# job does.
+- job-template:
+    <<: *e2e_job_defaults
+    name: 'continuous-docker-validation{suffix}'
```

Comment:
I too didn't like this. Started with a single job then realized it wouldn't give me the variable expansions in `e2e_job_defaults` so came up with this way.